gulpfile.js

```diff
@@ -17,6 +17,8 @@
             ' * ${pkg.description}\n' +
             ' * \n' +
             ' * @version ${pkg.version}\n' +
+            ' * @date ' + (new Date()).toISOString() + '\n' +
```

Comment:
Nah, it's not jsdoc, which describes the functionality (parameters, return type, etc). It just has a syntactic resemblance to jsdoc. `@link` is already there and isn't one either. I did notice a `@copyright` tag which I might use to identify RWAHS but I can make that a separate PR if we want it, this just gives us an easy way to verify deployments.